src/framework/theme/components/user/user.component.ts

```diff
@@ -118,11 +119,14 @@ export class NbUserComponent {
   @Input() title: string;
 
   /**
-   * Absolute path to a user picture
+   * Absolute path to a user picture. Or base64 image
    * User name initials (JD for John Doe) will be shown if no picture specified
    * @type string
    */
-  @Input() picture: string;
+  @Input() set picture(value: string) {
+    this.pictureValue = value;
+    this.backgroundImageStyle = this.domSanitizer.bypassSecurityTrustStyle(`url(${value})`);
```

Comment:
I guess we can do this without  additional `pictureValue` variable, 
`this.backgroundImageStyle = value ? this.domSanitizer.bypassSecurityTrustStyle(`url(${value})`) : null;`

and then in the template use this variable in `if` statements 
Also, I would personally prefer `imageBackgroundStyle` instead of `backgroundImageStyle` .